<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="nb-philipps-7" name="com.perfectomobile.StcTest" tests="2" failures="1" timestamp="19 Feb 2015 10:28:14 GMT" time="177,973" errors="0">
  <testcase name="f" time="95,254" classname="com.perfectomobile.StcTest">
    <failure type="java.lang.RuntimeException" message="Failed to close device 1E674EB8">
      <![CDATA[java.lang.RuntimeException: Failed to close device 1E674EB8
	at com.perfectomobile.selenium.MobileDevice.close(MobileDevice.java:117)
	at com.perfectomobile.selenium.MobileDevice.close(MobileDevice.java:107)
	at com.perfectomobile.StcTest.testCar2Go(StcTest.java:215)
	at com.perfectomobile.StcTest.f(StcTest.java:75)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:842)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1166)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1178)
	at org.testng.TestRunner.privateRun(TestRunner.java:757)
	at org.testng.TestRunner.run(TestRunner.java:608)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:36)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:368)
	at org.testng.internal.thread.ThreadUtil$CountDownLatchedRunnable.run(ThreadUtil.java:165)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
Caused by: com.perfectomobile.httpclient.HttpClientException: Failed to execute command handset close : null
	at com.perfectomobile.selenium.MobileEntity.processCommonResponse(MobileEntity.java:94)
	at com.perfectomobile.selenium.MobileDevice.processResponse(MobileDevice.java:504)
	at com.perfectomobile.selenium.MobileEntity.executeCommand(MobileEntity.java:58)
	at com.perfectomobile.selenium.MobileEntity.executeCommand(MobileEntity.java:50)
	at com.perfectomobile.selenium.MobileDevice.close(MobileDevice.java:115)
	... 23 more
]]>
    </failure>
  </testcase>
  <testcase name="f" time="82,719" classname="com.perfectomobile.StcTest"/>
</testsuite>
